tests/Tests/SKImageTest.cs

```diff
@@ -78,11 +79,8 @@ public void ReleaseImagePixelsWasInvoked()
 			using (var image = SKImage.FromPixels(pixmap, onRelease, "RELEASING!"))
 			{
 				Assert.False(image.IsTextureBacked);
-				using (var raster = image.ToRasterImage())
```

Comment:
This is no longer supported as it is not valid C#/.NET code. You have to actually dispose of the same `SKImage` instance twice (or more depending).If the instance is the same, you cannot dispose it more time.